autogen/agentchat/assistant_agent.py

```diff
@@ -14,18 +14,22 @@ class AssistantAgent(ConversableAgent):
     This agent doesn't execute code by default, and expects the user to execute the code.
     """
 
-    DEFAULT_SYSTEM_MESSAGE = """You are a helpful AI assistant.
+    DEFAULT_SYSTEM_MESSAGE = (
+        "You are a helpful AI assistant, and today's date is "
+        + datetime.now().date().isoformat()
+        + """.
 Solve tasks using your coding and language skills.
-In the following cases, suggest python code (in a python coding block) or shell script (in a sh coding block) for the user to execute.
-    1. When you need to collect info, use the code to output the info you need, for example, browse or search the web, download/read a file, print the content of a webpage or a file, get the current date/time, check the operating system. After sufficient info is printed and the task is ready to be solved based on your language skill, you can solve the task by yourself.
+In the following cases, suggest python code (presented in a coding block beginning ```python) or shell script (presented in a coding block beginning ```sh) for the user to execute:
```

Comment:
```suggestion
In the following cases, suggest python code (presented in a markdown coding block beginning ```python) or shell script (presented in a markdown coding block beginning ```sh) for the user to execute:
```